frontend/packages/schema-editor/src/components/SchemaInspector/ItemDataComponent.tsx

```diff
@@ -174,15 +173,9 @@ export function ItemDataComponent({ schemaNode }: IItemDataComponentProps) {
             </Switch>
           )}
           <ItemRestrictions schemaNode={schemaNode} />
-          <Divider marginless />
-        </>
-      )}
-      {hasCustomProps && (
-        <>
-          <CustomProperties path={schemaPointer} />
-          <Divider marginless />
         </>
       )}
+      {hasCustomProps && <CustomProperties path={schemaPointer} />}
```

Comment:
Will it be a divider between the rootPointer and the customProps now? Looks like this change only sets a divider in the top of the `FieldSet` below this component? 🤔

Reply:
Yes 😄 
<img width="493" alt="image" src="https://github.com/user-attachments/assets/335a3fdb-dafc-4d5e-b95b-ccf7780e99ef">
Inside `CustomProperties` there is another border